auto_nag/utils.py

```diff
@@ -46,7 +48,6 @@
     "enhancement": "S4",
 }
 
-
```

Comment:
Nit: unrelated newline removal

Reply:
Reverted in b118179bdda01a94051b543093c0adfe1d332cfc, it was just a cleanup in f26da577f24187aae4bbae7a86ad90b920a5756b